neocore/IO/BinaryReader.py

```diff
@@ -238,22 +228,27 @@ def ReadVarInt(self, max=sys.maxsize):
 
         Returns:
             int:
+
+        Raises:
+            ValueError: if the specified `max` number of bytes is exceeded
         """
-        fb = self.ReadByte()
-        if fb is 0:
-            return fb
+        try:
+            fb = self.ReadByte()
+        except ValueError:
+            return 0
         value = 0
-        if hex(fb) == '0xfd':
+        print(fb)
+        if fb == b'\xfd':
             value = self.ReadUInt16()
-        elif hex(fb) == '0xfe':
+        elif fb == b'\xfe':
             value = self.ReadUInt32()
-        elif hex(fb) == '0xff':
+        elif fb == b'\xff':
             value = self.ReadUInt64()
         else:
-            value = fb
+            value = int.from_bytes(fb, "little")
```

Comment:
did we previously have tests covering this branch? Just wondering if the output stays the same

Reply:
`make coverage` shows 100% coverage for `BinaryReader` as shown here.
![image](https://user-images.githubusercontent.com/40873301/53927218-25602800-4054-11e9-8eb0-b534bd2764b9.png)

I think all coverage comes from https://github.com/jseagrave21/neo-python-core/blob/2cf9ae2351a2565626c8553e2f529b8b5fccef35/tests/test_io.py#L95-L103